otx/algorithms/classification/configs/classification/efficientnet_b0_cls_incr/template.yaml

```diff
@@ -0,0 +1,62 @@
+# Description.
+model_template_id: Custom_Image_Classification_EfficinetNet-B0
+name: EfficientNet-B0
+task_type: CLASSIFICATION
+task_family: VISION
+instantiation: "CLASS"
+summary: Class-Incremental Image Classification for EfficientNet-B0
+application: ~
+
+# Algo backend.
+framework: OTEClassification v1.2.3
+
+# Task implementations.
+entrypoints:
+  base: otx.algorithms.classification.tasks.ClassificationTrainTask
+  openvino: otx.algorithms.classification.tasks.ClassificationOpenVINOTask
+  nncf: otx.algorithms.classification.adapters.deep_object_reid.tasks.ClassificationNNCFTask
+base_model_path: ../../../adapters/deep_object_reid/configs/classification/efficientnet_b0/template_experimental.yaml
```

Comment:
Is it difficult to combine two templates? There were no major problems with detection. (I have very little knowledge of deep-reid...)

Reply:
The configs for deep-object-reid is very different from mmcv, incurring errors that nncf requires to find deep-object-reid format config.

Reply:
mmdet and mmseg originally had the same structure, but there was nothing wrong with merging. Are there any errors?

Reply:
Yes, if you look main_model.py in dor folder, the configs are so different. and as far as I know, originally nncf requires a training to optimize the model, which means that it absolutely needs dor config. It is natural not to incur errors in det and seg cause they had the same structure.

Reply:
Yes, I understand. config would be good if so.
As I commented above, it seems good to merge the task classes for nncf. That seems to be no problem.